include/yaml-cpp/exceptions.h

```diff
@@ -7,6 +7,12 @@
 #pragma once
 #endif
 
+#ifdef _MSC_VER
+#define NOEXCEPT _NOEXCEPT
```

Comment:
how about just prefixing it with something that make it look less common? like `#define YAML_CPP_NOEXCEPT _NOEXCEPT`